doc/source/avl_ludwigsburg_de.md

```diff
@@ -0,0 +1,151 @@
+# AVL Ludwigsburg
+
+Support for schedules provided by [avl-ludwigsburg.de](https://www.avl-ludwigsburg.de) located in Baden Württemberg, Germany.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: avl_ludwigsburg
+      args:
+        city: Ludwigsburg
+        street: Bahnhofstraße
+```
+
+### Configuration Variables
+
+**city**<br>
+*(string) (required)*
+
+**street**<br>
+*(string) (optional - depending on city)*
+
+## Example
+
+```yaml
+waste_collection_schedule:
```

Comment:
Please remove the customize part.

Reply:
Fixed with 99f76ee391ace6bda6a79fd79bc9672f1e57c4a8